improvements to github exporter docs (#166)
port-labs/port-docs: docs/software-catalog/dashboards/dashboards.md

#### Visualization properties

| Field                   | Type     | Description                                                                                                                      | Default | Required |
| ----------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------- | ------- | -------- |

> | `Title`                 | `String` | The title of the visualization                                                                                                   | `null`  | `true`   |
> | `Icon`                  | `String` | The icon of the visualization                                                                                                    | `null`  | `false`  |
> | `Description`           | `String` | A short description to describe the visualization                                                                                | `null`  | `false`  |
> | `Blueprint`             | `String` | The Blueprint to visualize the data of its Related Entities                                                                      | `null`  | `true`   |
> | `Breakdown by property` | `String` | Group your chart by a specific property                                                                                          | `null`  | `true`   |
> | `Filters`               | `Array`  | Filters to include or exclue specific data based on Port's [Search Rules](../../software-catalog/search-in-port.md#search-rules) | []      | `false`  |

Reviewer: ```suggestion
| `Title`                 | `String` | Visualization title                                                                                                   | `null`  | `true`   |
| `Icon`                  | `String` | Visualization Icon                                                                                                     | `null`  | `false`  |
| `Description`           | `String` | Visualization description                                                                             | `null`  | `false`  |
| `Blueprint`             | `String` | The chosen Blueprint from which related Entities data is visualized from                                                                    | `null`  | `true`   |
| `Breakdown by property` | `String` | Group your chart by a specific property                                                                                          | `null`  | `true`   |
| `Filters`               | `Array`  | Filters to include or exclude specific data based on Port's [Search Rules](../../software-catalog/search-in-port.md#search-rules) | []      | `false`  |
```

> #### Example: Filter only `Deployment` entities from last week
> 
> Assuming we have a [Blueprint](../../software-catalog/blueprint/blueprint.md) that is called `Service` which is related to another called `Deployment` and we want to create visualizations on top of last week's deployments of this service.
> 
> To achieve this desired state, we can go into one of the `Service`'s specific page and create a new visualization and after selecting the `Deployment` Blueprint in the dropdown we can add the following filter to the the `Filters` array:

Reviewer: ```suggestion
#### Example: filter only `Deployment` Entities from last week

Let's assume we have a [Blueprint](../../software-catalog/blueprint/blueprint.md) that is called `Service` which is related to another Blueprint called `Deployment`, and we want to create visualizations on top of the last week's deployments of this service.

To achieve this desired state, we can go into one of the `Service`'s profile pages and create a new visualization. After selecting the `Deployment` Blueprint in the dropdown, we can add the following filter to the `Filters` array:
```
